doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -53,7 +53,7 @@ RF Front-End Modules
 Build system
 ------------
 
-|no_changes_yet_note|
+* Removed :kconfig:option:`CONFIG_MCUBOOT_IMAGE_VERSION` in favor of using a VERSION file, this version can be set via Kconfig by using :kconfig:option:`CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION`, but should be transitioned to using a dedicated :ref:`application VERSION file <zephyr:app-version-details>` instead.
```

Comment:
```suggestion
* Removed the :kconfig:option:`CONFIG_MCUBOOT_IMAGE_VERSION` Kconfig option in favor of using a dedicated :ref:`application VERSION file <zephyr:app-version-details>` file to set the version.
   You can alternatively set the version by using the :kconfig:option:`CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION` Kconfig option, but using a :file:`VERSION` file is the recommended approach.
```
I think that's still saying the same thing, just with a different focus.

Reply:
Updated